### DISCUSSION TITLE
Is there any way to increase the size of the logo?

### DISCUSSION THREAD

**Discussion by sardarms** [nunocoracao/blowfish#1231] [Q&A] [answered]
I want to increase the size of the logo. But I am unable to find the css that will allow me to do it. Can anyone please help?

**Comment by samuelwebb2** [comment]
To add to this. I am particularly interested in increasing the width. As there is an option to not show the title text in the header I would really appreciate if anyone could share how to adjust it so that we could have wider/larger logos. 
My societies logo is wide and has our society name in it thus we didn't need the option for title text in the header.

**Comment by sardarms** [comment]
For me, this works:

1. Create a custom.css file in the asset folder. [Here is the guide](https://blowfish.page/docs/advanced-customisation/#overriding-the-stylesheet)
2. Put the following code in the file:

```
.max-w-\[5rem\] {
    max-width: 10rem;
}
```

You can change the value in '10rem' as you like. 

Let me know if works for you.

**Reply by samuelwebb2** [reply, marked as answer]
Heya just wanted to come back to this and say thank you! This worked for me. 

For anyone else trying this I had to change the height instead of the width.

**Comment by shaflan** [comment]
The solution provided by @sardarms didn't work for me, however, it makes sense as the class that set the container of the logo to 80x80 is still (in newer versions, as of April 26) the "5rem". 
I asked the AI to modify this css code and it came with the following that worked for me:

span[class*="max-w-[5rem]"],
span[class*="w-[5rem]"],
.logo {
    width: 10rem !important;   
    max-width: none !important;
    height: auto !important;  
    max-height: none !important;
    display: inline-block !important;
}

/* Ensure the SVG inside fills the new width */
.logo svg {
    width: 100% !important;
    height: auto !important;
    max-width: none !important;
}
